qutip/tests/test_ipynbtools.py

```diff
@@ -0,0 +1,9 @@
+from qutip.ipynbtools import version_table
+import sys
+
+
+def test_version_table():
```

Comment:
Let's split this into two tests:

- `test_version_table` with is always called and sanity checks the version table.
- `test_version_table_with_cython` which is only called when Cython is installed.

E.g. 
```
@pytest.mark.parametrize('verbose', [False, True])
def test_version_table(verbose):
    html = version_table(verbose=verbose)
    assert  "<th>Software</th>" in html.data
    assert "<th>Version</th>" in html.data
    if verbose:
        assert "<td>Installation path</td>" in html.data


@pytest.importorskip("Cython")
def test_version_table_with_cython():
    html = version_table()
    assert "<td>Cython</td>" in html.data
```

Reply:
Hello @hodgestar,
Thanks for the review. 
Should I also check for other library dependencies under the `test_version_table` ?

Reply:
Yes, adding a few more checks to `test_version_table` would be good.